samples/python/requirements.txt

```diff
@@ -1,2 +1,5 @@
 opencv-python==4.5.*
 numpy>=1.16.6,<=1.23.1
+# For bert_becnmark
+transformers[onnx]
+torch
```

Comment:
It is also reasonable to require torch-cpu with is 10x less in size.

Reply:
There is no `torch-cpu`:
```sh
py -m pip install torch-cpu
ERROR: Could not find a version that satisfies the requirement torch-cpu (from versions: none)
ERROR: No matching distribution found for torch-cpu
```
I can see at https://pytorch.org/get-started/locally/ that for cuda support you need to add `--extra-index-url`. And cpu distribution is just `pip3 install torch torchvision torchaudio`

Reply:
Add the following to the `requirements.txt`:
```
--extra-index-url https://download.pytorch.org/whl/cpu
torch
torchvision
```

Reply:
this is ~180 Mb vs ~1.8 Gb considering CUDA python packages that come as dependencies of torch GPU distribution

Reply:
Here is what I see on Windows
```sh
py -m pip install torch
Collecting torch
  Using cached torch-1.13.0-cp39-cp39-win_amd64.whl (167.2 MB)
```
`torch` is even less than 180 Mb.

I don't think I need to add `--extra-index-url https://download.pytorch.org/whl/cpu`